packages/core/core/src/utils/transform-content-types-to-models.ts

```diff
@@ -13,17 +13,32 @@ const { identifiers } = utils;
  * Therefore we have to keep an additional set of helpers/extensions to the database naming methods
  */
 
-export const COMPONENT_JOIN_TABLE_SUFFIX = 'components';
-export const DZ_JOIN_TABLE_SUFFIX = 'components';
-export const COMPONENT_INVERSE_COLUMN_NAME = 'component';
-export const COMPONENT_TYPE_COLUMN = 'component_type';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const COMPONENT_JOIN_TABLE_SUFFIX = (options: MetadataOptions) => 'components';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const DZ_JOIN_TABLE_SUFFIX = (options: MetadataOptions) => 'components';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const COMPONENT_INVERSE_COLUMN_NAME = (options: MetadataOptions) => 'component';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const COMPONENT_TYPE_COLUMN = (options: MetadataOptions) => 'component_type';
+
 export const ENTITY = 'entity';
 
-export const getComponentJoinTableName = (collectionName: string) =>
-  identifiers.getTableName(collectionName, { suffix: COMPONENT_JOIN_TABLE_SUFFIX });
+// TODO: we should either support suffix with full+short, or call getNameFromTokens directly to get the correct values in unshortenedName map
```

Comment:
Same as above, this is handled by the other PR, correct?

Reply:
Same answer as above